package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.0.27",
+  "version": "2.0.29",
```

Comment:
should this be 28?

Reply:
I set 28 to the other PR, that's why this is 29

Reply:
just saw that, can we merge the 2 fixes in one PR?